pf9/modules/analytics_utils.py

```diff
@@ -0,0 +1,105 @@
+import analytics
+import uuid
+import math
+import time
+import datetime
+
+# the write_key is the Segment API authorization and identifier, without this no data submission will work. The API Key below is for Dev/Test work
+analytics.write_key = 't1tdSZocQ49Tx3G66lUjJXkOXjUSYRKx'
+
+def on_error(error, items):
+    print('An error occurred:', error)
+
+analytics.debug = True
+analytics.on_error = on_error
+
+
+class SegmentSessionWrapper:
+    def __init__(self, ctx):
+        self.ctx = ctx
+
+    def load_segment_session(self, segment_session, segment_event_properties, segment_subcommand):
+        """Load segment session related objects to ctx
+        """
+        self.ctx.params["segment_session"] = segment_session
+        self.ctx.params["segment_event_properties"] = segment_event_properties
+        self.ctx.params["segment_event_prefix"] = 'WZ PMKExpress CLI'
```

Comment:
pf9ctl CLI

Reply:
WZ PMKExpress CLI is what chris wanted it to be I think.

Reply:
I honestly want to get rid of Express. It is confusing. But we should consider this at some point in future.